src/RcppExports.cpp

```diff
@@ -1,6 +1,7 @@
 // This file was generated by Rcpp::compileAttributes
 // Generator token: 10BE3573-1514-4C36-9D1C-5A225CD40393
 
+#include "Rblpapi_types.h"
```

Comment:
That is odd.   Rcpp sometimes creates `NAMEOFPACKAGE_types.h` but it hasn't done that for me here.

And we don't have.  But we now have `rblpapi_types.h`.   I wonder if it got confused.